assets/translations/ge-GE.json

```diff
@@ -0,0 +1,62 @@
+{
+  "Create_new_account": "Neuen Account erstellen",
+  "Actions_History": "Aktionsverlauf",
+  "Empty_history": "Leere Geschichte",
+  "History_Settings_Page": "Seite Verlaufseinstellungen",
+  "Lamps_Page": "Lampen Seite",
+  "Name:_": "der Name: {}",
+  "Room:_": "Zimmer: {}",
+  "Lamps_Settings_Page": "Seite Lampeneinstellungen",
+  "Welcome_to_your_Smart_Home": "Willkommen in Ihrem Smart Home",
+  "Scene_Settings_Page": "Seite "Szeneneinstellungen"",
```

Comment:
You have quotation marks inside quotation marks.
Please removed the inside couple